### PR TITLE
Display results refactor

### LINKED ISSUE #24: Create Summary Component
Including:

- [ ] article title
- [ ] article URL
- [ ] article summary
- [ ] highlighted keywords

Response object will look like:
```
{
  'url':'http://bbc.co.uk',
  'article' : {
    'header' : 'An Article Title',
    'summary' : 'The Article Summary',
    'keywords' : ['Boris Johnson', 'Brexit']
  },
  'results' : [
    { 'type' : 'credibility' ...... },
    { 'type' : 'polarity' ..... },
    { 'type' : 'subjectivity' .....},
    { 'type' : 'biasscore' .....}
  ]
}
```

### LINKED ISSUE #26: Split 'CredibilityScore' from DisplayResult component (to it's own component)
Gauge to display:
- [ ] CredibilityScore returned from backend.
- [ ] Some explanation text

Response object will look like:
```
{
  'url':'http://bbc.co.uk',
  'article' : {
    'header' : 'An Article Title',
    'summary' : 'The Article Summary',
    'keywords' : ['Boris Johnson', 'Brexit']
  },
  'results' : [
    { 'type' : 'credibility' ...... },
    { 'type' : 'polarity' ..... },
    { 'type' : 'subjectivity' .....},
    { 'type' : 'biasscore' .....}
  ]
}
```

### PR DESCRIPTION
**Pull request by abirdonwire**
Breaking down of results, each piece of relevant data accessed and passed as props to the component that will render it inside display results.

### CODE REVIEW

File: src/components/Results/Credibility.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        </Grid>
+        <Grid item xs={12} md={4}>
+          <Typography variant="h5" className={classes.credibilityGaugeHeading}>
+            Credibility
+          </Typography>
+        </Grid>
+        <Grid item xs={12} md={4}>
+          <Typography variant="subtitle2" className={classes.categoryTitle}>
+            Category:{" "}
+            <Chip label={props.category} className={classes.categoryChip} />
```
Comment (jsjohnstone) [inline src/components/Results/Credibility.js]: Love this!!

File: src/components/Results/displayResults.js [hunk 2]
```diff
@@ -1,115 +1,65 @@
...
+      { type: "polarity", outcome: { score: 0.11155073461891642 } },
+      { type: "subjectivity", outcome: { score: 0.5150528007346189 } },
+    ],
+  };
+
+  const credibilityScore = resultsObj.results[0].outcome.score / 100;
   // accesses score from credibility analysis, assigns it to a variable,
   //the score is divided by 100 to provide the format required forthe percent prop on the gauge.
-  const source = resultsObj[0].source;
+  const category = resultsObj.results[0].outcome.category;
```
Comment (jsjohnstone) [inline src/components/Results/displayResults.js:14]: I've updated these in my PR as well - so hopefully they all merge together nicely 😬

**Review by jsjohnstone — APPROVED** [review]

File: src/components/Results/displayResults.js (hunk 2 above)
Comment (abirdonwire) [reply src/components/Results/displayResults.js:14]: I hope so! If not, it shouldn't be too difficult to resolve.

**Review by Jyothsna-GC — APPROVED** [review]

**Comment by abirdonwire** [comment]
I'm not going to merge yet, I'm scared!

**Pull request merged**